build-toolchain.bash

```diff
@@ -155,7 +160,7 @@ if [ $SKIP_THIRDPARTY != false ]; then
 		if [ ! -d binutils-build-ppc ]; then MISSING=true; fi
 		if [ ! -d gcc-build-ppc ]; then MISSING=true; fi
 	fi
-	if [ ! -d hfsutils ]; then MISSING=true; fi
+	if [ $SKIP_HFSUTILS = false -a ! -d hfsutils ]; then MISSING=true; fi
 
 	if [ $MISSING != false ]; then
 		echo "Not all third-party components have been built yet, ignoring --skip-thirdparty."
```

Comment:
```suggestion
		echo "Not all third-party components have been built yet; ignoring --skip-thirdparty and --skip-hfsutils."
		SKIP_HFSUTILS=false
```

Reply:
not sure this matches the original intention of the --skip-hfsutils flag.
After all, --skip-thirdparty means "I'm in a hurry and I believe these are already built" and --skip-hfsutils means "I already have hfsutils from some other source and I don't want to cause a mess".
Maybe --skip-hfsutils should be --with-system-hfsutils in the first place?

Reply:
> Maybe --skip-hfsutils should be --with-system-hfsutils in the first place?

I think you're right, that does make more sense.